CmwOverview.md

```diff
@@ -36,16 +35,14 @@ You receive a maintenance notification for an upcoming maintenance period. The c
 
 Note: 
 * The text "_You may initiate upgrade of your databases at any time_" distinguishes _Self-Service Maintenance_ notifications from other maintenance events.  The notification will also identify the subscription.
-* The deadline (2023-7-3 in this example) should allow at least 7 days to prepare. At any time prior to this deadline, you can initiate failover and complete the upgrade for a database, as described below. 
+* The deadline (2023-7-3 in this example) should allow at least 10 days to prepare. At any time prior to this deadline, you can initiate failover and complete the upgrade for a database, as described below. 
 
 ----
 
 # Step 2: You initiate failover to upgrade databases
 
-:exclamation: **Important:** _The process described below is appropriate if you wish to trigger failover for a small number of databases.
-For simplified failover of dozens or hundreds of databases, use [AzureSqlBulkFailover](AzureSqlBulkFailoverUsage.md) to trigger the failover 
-at the desired time. 
-
+:exclamation: **Important:** The process described below is appropriate if you wish to trigger failover for a small number of databases.
+For simplified failover of dozens or hundreds of SQL DB resources, you must [deploy AzureSqlBulkFailover](./AzureSqlBulkFailoverSetup.md) into the subscription that contains your databases and [use AzureSqlBulkFailover](AzureSqlBulkFailoverUsage.md) to trigger the failover at the desired time. Currently this solution doesn't support Azure SQL MI resources.
```

Comment:
This is incorrect. Perhaps it should read "For simplified failover of dozens or hundreds of SQL DB resources, you can [deploy AzureSqlBulkFailover](./AzureSqlBulkFailoverSetup.md) into a subscription and grant the automation account [access to](./AzureSqlBulkFailoverPermissions.md) the resource group that contains your databases and then use [use AzureSqlBulkFailover](AzureSqlBulkFailoverUsage.md) to trigger the failover at the desired time. Currently this solution doesn't support Azure SQL MI resources.